tools/gulp/tasks/lint.ts

```diff
@@ -46,6 +54,56 @@ task('madge', ['material:clean-build'], () => {
   });
 });
 
+task('ownerslint', () => {
+  let errors = 0;
+
+  let ownedPaths = readFileSync(ownersFilePath, 'utf8').split('\n')
+      // Trim lines.
+      .map(line => line.trim())
+      // Remove empty lines and comments.
+      .filter(line => line && !line.startsWith('#'))
```

Comment:
This won't handle cases like `/some/path owner1 owner2 #owner3`.

Reply:
I think it will? it splits on white space and just cares about the first piece